npm-js/create-kalix-entity/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "Lightbend Inc",
   "license": "Copyright (c) Lightbend Inc. 2021",
   "engines": {
-    "node": ">=14.0.0",
+    "node": ">=18.13.0",
```

Comment:
Should this be `>=18.0.0` or `~18`?

Reply:
Not sure how JS people usually do it. I guess ~18 is fine?

Reply:
Sounds good. That's been used in the other places.

Reply:
Done in [213245](https://github.com/lightbend/kalix-javascript-sdk/pull/459/commits/221324592055a1e63007d337bbcd6ca838792be7) 

We'll see what breaks. :)